### PR TITLE
fix const transpose

### PR DESCRIPTION
**Pull request by guschmue**

### CODE REVIEW

File: tf2onnx/tfonnx.py [hunk 1]
```diff
@@ -393,19 +391,30 @@ def conv_convert_inputs(ctx, node, with_kernel=False, new_kernel_shape=None,
...
-        transpose.set_attr("perm", HWCN_TO_NCHW)
-        transpose.inserted_nchw = True
-        transpose.skip_conversion = True
-        ctx.copy_shape(input_name, transpose.output[0])
-        new_shape = spatial_map(ctx.get_shape(input_name), HWCN_TO_NCHW)
-        ctx.set_shape(transpose.output[0], new_shape)
-        nodes.append(transpose)
-        parent.data_format = "NCHW"
+
+        need_transpose = True
```
Comment (pengwa) [inline tf2onnx/tfonnx.py]: why not use if- else, instead of need_transpose?

File: tf2onnx/tfonnx.py [hunk 2]
```diff
@@ -393,19 +391,30 @@ def conv_convert_inputs(ctx, node, with_kernel=False, new_kernel_shape=None,
...
+        if node.inputs[1].is_const():
+            # kernel is const - transpose the const if we are the only consumer of const
+            # TODO: maybe we should make a copy of the const, or look at the other consumers
+            # if they'd want a transose as well.
```
Comment (pengwa) [inline tf2onnx/tfonnx.py]: a typo - transose

File: tf2onnx/tfonnx.py (hunk 1 above)
Comment (guschmue) [reply tf2onnx/tfonnx.py]: ah, because there are 2 conditions ... if cons and if consumers==1

Comment (pengwa) [reply tf2onnx/tfonnx.py]: sorry...misunderstood

File: tf2onnx/tfonnx.py (hunk 2 above)
Comment (guschmue) [reply tf2onnx/tfonnx.py]: ah, will fix with next commit

**Comment by nbcsm** [comment]
@guschmue, seems the CLA is not signed?

**Pull request merged**